blueman/gui/manager/ManagerDeviceMenu.py

```diff
@@ -213,6 +252,32 @@ def Generate(self):
             self.append(item)
             return
 
+        # Generic (dis)connect
+        # LE devices do not appear to expose certain properties like uuids until connect to at least once.
+        # show generic connect for these as we will not show any way to connect otherwise.
+        device_uuids = self.SelectedDevice['UUIDs']
+        show_generic_connect = not device_uuids
+        for uuid in device_uuids:
+            if ServiceUUID(uuid).short_uuid in (
+                AUDIO_SOURCE_SVCLASS_ID, AUDIO_SINK_SVCLASS_ID, GENERIC_ACCESS_SVCLASS_ID,
+                HANDSFREE_AGW_SVCLASS_ID, HEADSET_SVCLASS_ID, HID_SVCLASS_ID
```

Comment:
We should drop `GENERIC_ACCESS_SVCLASS_ID`. We're only interested in 0x1812, not GATT in general.